### ISSUE TITLE
G82 Technique title (SC 1.1.1)

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/wcag#2118]
I created this issue on behalf of the [Alt Text Subgroup] (https://www.w3.org/WAI/GL/task-forces/silver/wiki/Alt_Text_Subgroup ) of the Silver Task Force. 

### Explanation
The title for the G82 technique reads "[G82: Providing a text alternative that identifies the purpose of the non-text content.](https://www.w3.org/WAI/WCAG21/Techniques/general/G82)"
 

This is a general technique for "Situation C: If non-text content is a control or accepts user input." The wording is ambiguous. For instance, it’s almost the same as the G94 title which reads "G94: Providing short text alternative for non-text content that serves the same purpose and presents the same information as the non-text content."

"Situation C" specifically addresses interactive UI components such as form controls, image maps, links, submit buttons, and so on. 

### Recommendation
The Alt Text Subgroup proposes changing the G82 technique title to read:
"Providing a text alternative that identifies the purpose of **interactive** non-text content." 

The word "interactive" was added to make it less ambiguous.

**Comment by mbgower** [comment]
@MakotoUeki 
In the Understanding for 1.1.1, G82 continues to be listed as the only general technique for Situation C; however, it now contains the following qualifier:
> [G82: Providing a text alternative that identifies the purpose of the non-text content](https://www.w3.org/WAI/WCAG22/Techniques/general/G82) using one of the following [Text alternative techniques for controls and input for Situation C](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content#text-equiv-all-situation-c-controls)

We believe this addresses your concern. If you feel this is addressed, please close the issue.

**Comment by kfranqueiro** [comment]
I disagree with @mbgower, as his response only addresses the concern in the context of the link _to_ G82 within the Understanding page for non-text content, whereas this issue is referring to the title and h1 within the G82 page itself.

Furthermore, @MakotoUeki's point regarding potential ambiguity between G82 and G94 would also apply to the [techniques index page](https://www.w3.org/WAI/WCAG22/Techniques/).

This came up in discussion during the Backlog TF meeting on 21 Nov 2025; Mike expressed concern that "interactive" might not fit the content of G82, but in my reading of it, the entire context of the technique is specific to providing a text alternative for interactive components that perform a function. As far as I can tell, the original suggestion from the alt text subgroup seems like it would work. I can create a PR.
